app/scripts/modules/kubernetes/src/v2/pipelines/stages/patchManifest/patchManifestConfig.html

```diff
@@ -0,0 +1,55 @@
+<div ng-if="ctrl.state.loaded" class="clearfix">
+  <div class="alert alert-warning">
+    <strong>Warning!</strong> This stage is under active development and is subject to change.
+  </div>
+
+  <div class="container-fluid form-horizontal">
+    <h4>Resource to Patch</h4>
+    <div class="horizontal-rule"></div>
+    <kubernetes-manifest-selector selector="ctrl.$scope.stage"></kubernetes-manifest-selector>
+
+    <h4>Patch Content</h4>
+    <hr>
+    <ng-form name="kubernetesManifestSource">
+      <stage-config-field label="Manifest Source" help-key="kubernetes.manifest.source">
```

Comment:
Could this be `Patch Content Source` (since it seems like you've otherwise mostly swapped `patch content` for `manifest`)